### PR TITLE
[BACKLOG-29564] - Karaf assembly pme ce

### PR DESCRIPTION
**Pull request by Pancho7**
@pentaho/millenniumfalcon

### CODE REVIEW

File: pentaho-karaf-assembly/pom.xml.orig [hunk 1]
```diff
@@ -0,0 +1,369 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (davidmsantos90) [inline pentaho-karaf-assembly/pom.xml.orig]: this file seems to have been committed by mistake

File: assemblies/pme/pom.xml [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+          <execution>
+            <id>default-assembly</id>
+            <configuration>
+              <bootFeatures>
+                <feature>config</feature>
+                <feature>pentaho-client-minimal</feature>
+                <feature>pentaho-fasterxml</feature>
+                <feature>pentaho-dataservice-client</feature>
+                <feature>community-edition</feature>
+                <feature>package</feature>
```
Comment (ricardosilva88) [inline assemblies/pme/pom.xml]: Is package real needed? Isn't already included on startup properties?

File: assemblies/pme/pom.xml [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+              </bootFeatures>
+              <installedFeatures>
+                <feature>pentaho-karaf-overrides</feature>
```
Comment (ricardosilva88) [inline assemblies/pme/pom.xml]: For now, add "ssh" feature to installed feature. following the same approach we did for pentaho server - https://github.com/pentaho/pentaho-karaf-assembly/pull/563/files#diff-d53ef37ce59237a260e596101a96f9e5R168
By being installed, it doesn't mean that will be enabled, but at least we guarantee that will be present on karaf

File: assemblies/pme/pom.xml (hunk 2 above)
Comment (Pancho7) [reply assemblies/pme/pom.xml]: package is necessary to run commands in ssh mode

File: assemblies/pme/pom.xml (hunk 3 above)
Comment (Pancho7) [reply assemblies/pme/pom.xml]: Done

File: assemblies/pme/src/assembly/assembly.xml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  ~
+  ~ Copyright 2014 - 2019 Hitachi Vantara. All rights reserved.
+  -->
+
+<assembly  xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.3"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.3 http://maven.apache.org/xsd/assembly-1.1.3.xsd">
+
+
+<id>pme</id>
```
Comment (nantunes) [inline assemblies/pme/src/assembly/assembly.xml]: Bad indentation.

File: assemblies/pme/src/main/resources/config/README [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ *
+ * Copyright (c) 2019 Pentaho Corporation. All rights reserved.
+ */
+
+This folder is intended to be used to deploy into the Karaf container exploded
+artifacts with configurations.
```
Comment (nantunes) [inline assemblies/pme/src/main/resources/config/README]: What?

File: pom.xml [hunk 6]
```diff
@@ -143,6 +143,14 @@
         <type>xml</type>
         <scope>runtime</scope>
       </dependency>
+      <dependency>
+        <groupId>pentaho</groupId>
+        <artifactId>pentaho-karaf-features</artifactId>
+        <classifier>standard</classifier>
```
Comment (nantunes) [inline pom.xml]: Wasn't this replaced with org.hitachivantara.karaf.features/pentaho-karaf-features-standard, already included above?

Comment (Pancho7) [reply pom.xml]: Fixed

File: assemblies/pme/src/main/resources/config/README (hunk 5 above)
Comment (Pancho7) [reply assemblies/pme/src/main/resources/config/README]: Config folder not necessary. Deleted

File: assemblies/pme/src/assembly/assembly.xml (hunk 4 above)
Comment (Pancho7) [reply assemblies/pme/src/assembly/assembly.xml]: Done

**Pull request merged**